package/gluon-ebtables-net-rules/files/lib/gluon/ebtables/110-net-allow-ipv6

```diff
@@ -0,0 +1,8 @@
+siteConfig = require("gluon.site_config")
+
+rule ('NET_SRC -p IPv6 --ip6-src fe80::/10 -j RETURN')
```

Comment:
For IPv6 Duplicate Address Detection "::" is used as the source address of an ICMPv6 Neighbor Solicitation message. This is mandatory by the IPv6 standard.

(Could it be that this is where your 30% overhead reduction came from?)